src/index.ts

```diff
@@ -89,7 +90,7 @@ async function retryFetch(
         const res = await fetch(requestUrl, requestOptions);
         if (shouldRetry(res)) throwRetryError(null);
         jsonFetchOptions.onRequestEnd?.({
-          responseOrError: res,
+          status: res,
```

Comment:
We just want the status. That's the main problem we're trying to solve with this PR: the response is too huge to be logged!
```suggestion
          status: res.status,
```